lib/rules/arrow-parens.js

```diff
@@ -51,13 +51,32 @@ module.exports = {
         const sourceCode = context.getSourceCode();
 
 
+
         /**
          * Determines whether a arrow function argument end with `)`
          * @param {ASTNode} node The arrow function node.
          * @returns {void}
          */
         function parens(node) {
-            const token = sourceCode.getFirstToken(node, node.async ? 1 : 0);
+            const isAsync = node.async;
+            const token = sourceCode.getFirstToken(node, isAsync ? 1 : 0);
```

Comment:
I know this was here before, but can you give this variable a more descriptive name? It's not clear which token it's referring to.